### ISSUE TITLE
Review homepage for accessibility

### ISSUE THREAD

**Issue by sshetenhelm** [yalelibrary/YUL-DC#3068]
**Story**
We want to ensure DCS is accessible for all users. As per [Yale’s Web Accessibility Policy](https://your.yale.edu/policies-procedures/policies/1605-web-accessibility-policy), “University Websites should be in compliance with the most recent version of or successor standards to the [Web Content Accessibility Guidelines (“WCAG”) 2.0 Level AA](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.0), as published by the Web Accessibility Initiative of the World Wide Web Consortium.”

Please review the Blacklight homepage at https://collections.library.yale.edu/ and identify any accessibility issues, including problems with navigation, readability, and usability.

Please provide recommendations for fixes.

For your reference — Web Accessibility Testing Procedures (https://usability.yale.edu/web-accessibility/articles/testing-procedures) from Usability & Web Accessibility (https://usability.yale.edu/)

**Acceptance**
- [ ] Review homepage for issues
- [ ] List issues in this ticket
- [ ] Suggest fixes for each issue

**Comment by tmacmath** [comment]
**HOME PAGE:** 

**Issues:**
- Small text below "About" h2 and "Help" h2
- Small text: image caption
- Link text missing context: "More information on additional collections can be found here."

**Fixes:** 
- Change small font size to 16px
- Change image caption font size to .875rem (14px)
- Change link text to "More information on additional collections can be found on the Digital Collections Page."

**Comment by tmacmath** [comment]
**ADVANCED SEARCH PAGE:** 

**Issues:** 

- Page does not start with a level 1 heading: "Advanced Search" is a level 1 heading, but the heading that is being read first is the h2 in the nav bar. 
- Ideally, the Yale wordmark should not be an image.
- Broken ARIA reference: all/any sort field, all search form fields (An element has an aria-labelledby or aria-describedby value that does not match the id attribute value of another element in the page)

**Fixes:** 

- Headings should ONLY be used for page structure, and not for styling. Use <ul> for navbar list items and style with CSS. 
- Add alt text to the functional image/link as described here: https://www.w3.org/WAI/tutorials/images/functional/
Example: alt="Yale Library Home"
-  Do not use "arialabelled-by" for form elements; use <label> instead: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-labelledby

Examples: 

<img width="1029" height="455" alt="Image" src="https://github.com/user-attachments/assets/8a80e118-5822-47c4-a19d-bf47b198cab6" />


If using "aria-labelledby" in the future, fix the typo. In our code it is "aria-labeledby", which is not a valid element.

**Comment by tmacmath** [comment]
**PARENT OBJECT PAGE:** 

**Issues:** 

None. For some reason, all of the audit tools flagged the link color as low contrast, but the WebAim Color Contrast Checker gave it a Pass for WCAG AA. 

**Fixes:** 

None.
